cli/module_registry.go

```diff
@@ -78,7 +79,8 @@ type moduleManifest struct {
 	Models      []ModuleComponent `json:"models"`
 	// JsonManifest provides fields shared with RDK proper.
 	modconfig.JSONManifest
-	Build *manifestBuildInfo `json:"build,omitempty"`
+	Build     *manifestBuildInfo `json:"build,omitempty"`
+	Overrides map[string]any     `json:"overrides"`
```

Comment:
```
➜  tmp git:(meta-platform-overrides) ✗ viam module create --public-namespace foo --name baz --local-only
Configuration for the module has been written to meta.json
➜  tmp git:(meta-platform-overrides) ✗ cat meta.json
{
  "$schema": "https://dl.viam.dev/module.schema.json",
  "module_id": "foo:baz",
  "visibility": "private",
  "url": "",
  "description": "",
  "models": [
    {
      "api": "",
      "model": ""
    }
  ],
  "entrypoint": "",
  "overrides": null
}%                                                                                                                                                                                         
```

Suggestion:

Either add omitempty to this or set the default to an empty map in the creation logic. (so that it is not `null`)